lib/plugin/instance.rb

```diff
@@ -516,6 +516,18 @@ def after_initialize(&block)
     initializers << block
   end
 
+  def commit_hash
+    @commit_hash ||= git_repo.latest_local_commit
+  end
+
+  def commit_url
+    @commit_url ||= "#{git_repo.url}/commit/#{git_repo.latest_local_commit}"
```

Comment:
I see that we're already memoizing stuff stuff in the `GitRepo` instance. Is there a need to memoize here again?